### PR TITLE
fix failing tests

### PR DESCRIPTION
**Pull request by ocefpaf**
I want to send another PR with some code changes but I noticed that some of the tests are stale and failing. We should mock most of those but I'm not sure if it is worth fixing this library, it seems to be on life support already.

### CODE REVIEW

File: tests/collectors/test_coops_sos.py [hunk 1]
```diff
@@ -31,23 +31,19 @@ def test_coops_describe_sensor(self):
         assert isinstance(response[0], SensorML)
 
     def test_raw_coops_get_observation(self):
-        self.c.start_time   = datetime.strptime("2012-10-01", "%Y-%m-%d")
-        self.c.end_time     = datetime.strptime("2012-10-02", "%Y-%m-%d")
-        self.c.features     = ['8454000']
```
Comment (ocefpaf) [inline tests/collectors/test_coops_sos.py]: This was returning

```
owslib.ows.ExceptionReport: 'There is no Water Level sensor installed at station
urn:ioos:station:NOAA.NOS.CO-OPS:8454000 or dissemination has been stopped by CORMS.'
```

so I changed to another station.

File: tests/collectors/test_ioos_swe_sos.py [hunk 2]
```diff
@@ -19,19 +21,22 @@ def setUp(self):
         # self.c.features = [urn.name for urn in self.c.server.offerings]
 
 
+    @pytest.mark.xfail
```
Comment (ocefpaf) [inline tests/collectors/test_ioos_swe_sos.py]: The endpoint http://data.nanoos.org/52nsos/sos/kvp is timing out (`504 Gateway Time-out`). Ping @emiliom who may know that is going on here. Skipping the tests for now...

File: tests/collectors/test_usgs.py [hunk 3]
```diff
@@ -52,8 +52,8 @@ def test_by_state(self):
         assert station.name == "TEN MILE R., PAWTUCKET AVE. AT E. PROVIDENCE, RI"
         # Measures 2 variables
         assert len(station.get_unique_members()) == 2
-        assert station.location.x == -71.3511649
-        assert station.location.y == 41.83093376
+        assert station.location.x == -71.3503315
+        assert station.location.y == 41.83093377
```
Comment (ocefpaf) [inline tests/collectors/test_usgs.py]: The position changed.

**Comment by ocefpaf** [comment]
I no longer have merge powers on IOOS repositories (ping @mwengren). Can you please merge this one @Bobfrat?

**Comment by Bobfrat** [comment]
I dont have privileges either

**Comment by ocefpaf** [comment]
> I dont have privileges either

Thanks! @mwengren I guess you are the only one that can merge this now.

**Comment by mwengren** [comment]
@rsignell-usgs @ocefpaf Maintaining teams is the right way to do this.  We actually do this extensively already, it's just that you both were org admins which grants the rights to do anything with the IOOS org, including delete it.  I've already added both of you to the team that has write access to pyoos.

No ill will intended, but it's in our best interest (if not yet required by policy  - NOAA or otherwise) to limit the risks of losing our data.

**Comment by rsignell-usgs** [comment]
@mwengren , okay makes sense.

**Comment by ocefpaf** [comment]
Thanks @mwengren for restoring our merge rights, @Bobfrat for the review, and @rsignell-usgs for the merge!

File: tests/collectors/test_ioos_swe_sos.py (hunk 2 above)
Comment (emiliom) [reply tests/collectors/test_ioos_swe_sos.py]: We had a problem this morning with a server involved in the SOS service. It's fixed. 

But more generally, maybe this test should be changed to point to a different SOS service?

Comment (ocefpaf) [reply tests/collectors/test_ioos_swe_sos.py]: Thanks for the info @emiliom. In theory all the unittest should be mocked and not relying on remote services, only integration tests should rely on those and should be `xfail`. However, that is a considerable amount of work and I'm not sure we can afford to do that for `pyoos` (unless there is a community interest to keep this library alive).

**Pull request merged**